core/src/main/scala/io/snappydata/SnappyDaemons.scala

```diff
@@ -50,25 +50,22 @@ object SnappyAnalyticsService extends Logging {
 				.getOrElse(DEFAULT_ANALYTICS_SERVICE_INTERVAL).toString
 		connProperties =
 				ExternalStoreUtils.validateAndGetAllProps(sc, mutable.Map.empty[String, String])
-		getScheduledExecutor.scheduleWithFixedDelay(
-			getTotalMemoryUsagePerTable, ZERO,
-			delayInMillisconds.toLong, TimeUnit.MILLISECONDS)
-	}
+		if (analyticsExecutor == null || analyticsExecutor.isShutdown) {
+			analyticsExecutor =	newDaemonSingleThreadScheduledExecutor("SnappyAnalyticsService")
+			analyticsExecutor.scheduleWithFixedDelay(
+				getTotalMemoryUsagePerTable, ZERO,
+				delayInMillisconds.toLong, TimeUnit.MILLISECONDS)
+		}
```

Comment:
SNAP-733 is opened. for now setting  DEFAULT_ANALYTICS_SERVICE_INTERVAL to 10 seconds